### ISSUE TITLE
Several crucial recipes missing [2.1.6]

### ISSUE THREAD

**Issue by ghost** [Alex-the-666/Ice_and_Fire#3927]
**Describe the bug**
Some recipes are missing like the dragon meal, i can't even craft it yet, i can obtain it with creative mode? however i can craft the sickly dragon meal, also the serpent scales armour set can't be crafted, i haven't seen what else might be missing but it is rather bothersome

**To Reproduce**
1. Go to 'crafting table'
2. Click on 'crafting table
3. See error

**Expected behavior**
Be able to see, interact and use the recipe and be able to craft it

**Screenshots**
![2021-06-04_16 58 57](https://user-images.githubusercontent.com/73154515/120866803-45923080-c556-11eb-83ca-d7e95822af30.png)
![2021-06-04_16 57 47](https://user-images.githubusercontent.com/73154515/120866806-462ac700-c556-11eb-8dbe-cf1c57fb4774.png)

**Additional context**
At first I thought it was a block limit problem however even when i delete the decoration mods i have it doesn't seem to work? i tried reinstalling minecraft and still nothing, I even installed JEI to see if that would fix it yet it just makes it weirder?? is there a way it can be fixed without making a whole new world from scratch?

**Comment by FlameyFaucet** [comment]
The vanilla recipe book is incredibly unsuited to modded content. It won't try and often will not ever show modded recipes. But hey, you have JEI installed it seems. That works too. Simply either press the show recipe keybind over the item you want to craft or click it (without cheat mode enabled). There's a small plus button there; that's the fill recipe button. Will automatically distribute your items into the recipe shape.
tl;dr use JEI lol recipe book sucks
tl;dr 2 you can always just look up the recipe elsewhere and arrange it yourself

**Comment by ghost** [comment]
> The vanilla recipe book is incredibly unsuited to modded content. It won't try and often will not ever show modded recipes. But hey, you have JEI installed it seems. That works too. Simply either press the show recipe keybind over the item you want to craft or click it (without cheat mode enabled). There's a small plus button there; that's the fill recipe button. Will automatically distribute your items into the recipe shape.
> tl;dr use JEI lol recipe book sucks
> tl;dr 2 you can always just look up the recipe elsewhere and arrange it yourself

I actually tried this yet i can't craft it, i do know how to craft the dragon meal it is very simple, is just that no matter what i do even if i put the items like i'm supposed to, the dragon meal dosen't even appear, i can't craft it according to JEI and even with new worlds dosen't seem to work either, i've tried reinstalling minecraft and redownloading my mods, yet the issue persists only with ice and fire though.

**Comment by ghost** [comment]
NEVERMIND FIXED IT!
I deleted a few files in the .minecraft file and it seems that worked, it is not a permanent solution however the files might corrupt again but as of now is all i needed, thank you @FlameyFaucet for the help though! gonna close this later i'm kinda busy atm.
